webclient/app/src/app/commons/appTemplateDialog/AppTemplateDialog.jsx

```diff
@@ -12,31 +12,54 @@ import {
 import AppTemplateDialogStyles from "./AppTemplateDialogStyles";
 import { useTranslation } from "react-i18next";
 import { Close } from "@mui/icons-material";
+import AppTemplateRest from "../../services/AppTemplateRest"
+import ErrorAlert from "../alert/ErrorAlert";
 
 function AppTemplateDialog(props) {
-
-    const appTemplateDialogStyles = AppTemplateDialogStyles();
+    const { appTemplate, open, onClose, onRefresh, isCreateDialog } = props;
     const { t } = useTranslation();
     const [internalAppTemplate, setInternalAppTemplate] = useState(null);
+    const [alert, setAlert] = useState({"open":false, "title": "ERROR", "message": ""});
+
+    const appTemplateDialogStyles = AppTemplateDialogStyles();
+    const appTemplateRest = new AppTemplateRest();
 
-    const { appTemplate, open, onClose, onRefresh, isCreateDialog } = props;
 
     const onDialogClose = () => {
         onClose();
+        closeAlert();
         setInternalAppTemplate(appTemplate);
     }
 
-    function handleChange(event) {
+    const handleChange = (event) => {
         const { name, value } = event.target;
         let appTemplateNew = { ...internalAppTemplate };
         appTemplateNew[name] = value;
         setInternalAppTemplate(appTemplateNew);
     }
 
-    function handleSave(toSave) {
-        // TODO: Add send to server
-        const savedAppTempate = toSave;
-        setInternalAppTemplate(savedAppTempate);
+    const handleSave = (toSave) => {
+        if (isCreateDialog) {
+            appTemplateRest.create(toSave).then(response => {
+                handleSaveResponse(response);
+            }).catch(err => {
+                return setAlert({"open":true, "title": "alert.error", "message": JSON.stringify(err.response.data)});
+            });
+        } else {
+            appTemplateRest.update(toSave).then(response => {
+                handleSaveResponse(response);
+            }).catch(err => {
+                return setAlert({"open":true, "title": "alert.error", "message": JSON.stringify(err.response.data)});
```

Comment:
I think this ``message`` would be great in a technical information panel with an extra prop. But I like that there is more information than less.